Postman/Postman.js

```diff
@@ -107,111 +101,108 @@ Seriesコレクションから新刊のBookListドキュメントを抽出する
-	}
-	if(newReleasies.length === 0){
-		return done({
-			"status": 'not-sent',
-			"message": "通知するシリーズ書籍がありません"
-		});
-	}
-	done(null, newReleasies);
+Postman.prototype.inspectSeries = function inspectSeries(seriesItems, done) {
+  let index;
+  let seriesItem;
+  let newReleasies = [];
+  for (index = 0; index < seriesItems.length; index++) {
+    seriesItem = seriesItems[index];
+    newReleasies = this._diffItems(seriesItem.recentContains, seriesItem.currentContains);
+  }
+
+  if (newReleasies.length === 0) {
+    return done({
+      'status': 'not-sent',
+      'message': '通知するシリーズ書籍がありません',
+    });
+  }
+  done(null, newReleasies);
 };
 
 
-/**
-**/
-Postman.prototype._diffItems = function(srcArray, criteriaArray){
-	var diffArray = _.filter(srcArray, function(obj){
-		return !_.findWhere(criteriaArray, obj);
-	});
-	return diffArray;
+Postman.prototype._diffItems = function _diffItems(srcArray, criteriaArray) {
+  const diffArray = _.filter(srcArray, (obj)=> {
+    return !_.findWhere(criteriaArray, obj);
+  });
+  return diffArray;
 };
 
 
-Postman.prototype.run = function(){
-	var _fetch = this._defer(this.fetch.bind(this));
-	var _sent = this._defer(this.sent.bind(this));
+Postman.prototype.run = function run() {
+  const _fetch = this._defer(this.fetch.bind(this));
+  const _sent = this._defer(this.sent.bind(this));
 
-	Q.when()
-	.then(_fetch)
-	.then(_sent)
-	.then(function(){
-		log.info('全メールの配信が完了');
-	})
-	.fail(function(err){
-		log.info(err);
-	});
+  Q.when()
+  .then(_fetch)
+  .then(_sent)
+  .then(()=> {
+    log.info('全メールの配信が完了');
+  })
+  .fail((err)=> {
+    log.info(err);
+  });
 };
 
-
-
-Postman.prototype.sentSeries = function(user){
-	var args = Array.prototype.slice.call(arguments);
-			args = _.compact(args);
-	var done = args[args.length - 1];
-
-	var _Mailer = this.Mailer;
-
-	var _fetchSeries = this.Utils.defer(this.fetchSeries.bind(this));
-	var _inspectSeries = this.Utils.defer(this.inspectSeries.bind(this));
-
-	var _createTemplate = this.Utils.defer(function(series, done){
-		_Mailer.createTemplate('series', series, function(err, mailStrings){
-			if(err){
-				return done(err);
-			}
-			_Mailer.setMail("info@kindlized.it", user.mail, "[kindlize.it] 新刊通知", mailStrings);
-			done();
-		});
-	});
-
-	var _sendMail = this.Utils.defer(_Mailer.send.bind(_Mailer));
-
-	Q.when(user)
-	.then(_fetchSeries)
-	.then(_inspectSeries)
-	.then(_createTemplate)
-	.then(_sendMail)
-	.then(function(series){
-		done(null, series);
-	})
-	.fail(function(err){
-		done(null, err);
-	});
+Postman.prototype.sentSeries = function sentSeries(user) {
+  let args = Array.prototype.slice.call(arguments);
+  args = _.compact(args);
+
+  const done = args[args.length - 1];
+  const _Mailer = this.Mailer;
+
+  const _fetchSeries = this.Utils.defer(this.fetchSeries.bind(this));
+  const _inspectSeries = this.Utils.defer(this.inspectSeries.bind(this));
+
+  const _createTemplate = this.Utils.defer((series, callback)=> {
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).

*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).